BierBuddy.Tests/DataAccessTests.cs

```diff
@@ -0,0 +1,259 @@
+            _conn.Open();
+
+            // Run alle sql bestanden in een folder
+            string[] files = Directory.GetFiles("../../../../BierBuddy/migrations", "V*.sql");
+            foreach (string file in files)
+            {
+                // Lees de SQL-query uit het bestand
+                string sqlQuery = File.ReadAllText(file);
+
+                // Voer de query uit
+                using (MySqlCommand cmd = new MySqlCommand(sqlQuery, _conn))
+                {
+                    cmd.ExecuteScalar();
+                }
+                Thread.Sleep(1000); // Wacht 1 seconde
+
+            }
+            _conn.Close();
+            string testConnectionString = "server=localhost;database=test_db;user=root;port=3306;password=";
+            _conn = new MySqlConnection(testConnectionString);
+            _conn.Open();
+            using (var command = new MySqlCommand($"SHOW DATABASES LIKE 'test_db';", _conn))
+            {
+                using (var reader = command.ExecuteReader())
+                {
+                    bool has = reader.HasRows;
+                }
+            }
+        }
+
+        [Test]
+        [Order(0)]
+        public void AddAccount_ShouldThrowArgumentException_WhenInvalidNumberOfItems()
+        {
+            MySQLDatabase database = new MySQLDatabase(_conn);
+
+            // Act & Assert
+            Assert.Throws<ArgumentException>(() => database.AddAccount("Test", "Bio", 30, new List<long>(), new List<long> { 1, 2, 3, 4 }, new List<long> { 1, 2, 3, 4 }, new List<string> { "a", "b", "c", "d" }));
+        }
+
+        [Test]
+        [Order(1)]
+        public void AddAccount_ShouldAddAccount_WhenValidNumberOfItems()
+        {
+            MySQLDatabase database = new MySQLDatabase(_conn);
+
+            // Act
+            Visitor? account = database.AddAccount("Test", "Bio", 30, new List<long> { 1, 2, 3, 4 }, new List<long> { 1, 2, 3, 4 }, new List<long> { 1, 2, 3, 4 }, new List<string> { "a", "b", "c", "d" });
+
+            // Assert
+            Assert.IsNotNull(account);
+        }
+
+        [Test]
+        [Order(2)]
+        public void GetAccount_ShouldReturnAccount_WhenAccountExists()
+        {
+            MySQLDatabase database = new MySQLDatabase(_conn);
+
+            // Act
+            Visitor? account = database.GetAccount(1);
+
+            // Assert
+            Assert.IsNotNull(account);
+        }
+
+        [Test]
+        [Order(3)]
+        public void GetAccount_ShouldReturnNull_WhenAccountDoesNotExist()
+        {
+            MySQLDatabase database = new MySQLDatabase(_conn);
+
+            // Act
+            Visitor? account = database.GetAccount(2);
+
+            // Assert
+            Assert.IsNull(account);
+        }
+
+        [Test]
+        [Order(4)]
+        public void AddAccount_ShouldAddAccount_WhenValid()
+        {
+            MySQLDatabase database = new MySQLDatabase(_conn);
+
+            // Act
```

Comment:
comments moeten in het nederlands

Reply:
Code termen houden vanaf nu in het Engels